server/lobby/ServerLobby.gd

```diff
@@ -67,7 +67,7 @@ func get_name() -> String:
 # Parse command line port in the form of:
 # --port xxxxx
 func get_port() -> int:
-	var port := ServerNetwork.SERVER_PORT
```

Comment:
why can't this be strongly typed?

Reply:
I don't know actually, it was a seemingly unrelated error message I got while debugging.  I am actually not sure if it's already a bug even without my changes - I imagine it must be.  But I was seeing this error come up when i was resolving my other errors.